hezar/models/image2text/beit_roberta/beit_roberta_image2text.py

```diff
@@ -65,14 +64,18 @@ def forward(
             encoder_outputs=encoder_outputs,
             past_key_values=past_key_values,
             decoder_inputs_embeds=decoder_inputs_embeds,
-            labels=labels,
             use_cache=use_cache,
```

Comment:
It's better to explicitly set labels to None! So code reviewers will get our approach.

Reply:
@pooya-mohammadi Nice catch. Done.